src/modules/Elsa.Liquid/Extensions/TemplateContextExtensions.cs

```diff
@@ -18,5 +19,13 @@ internal static void AddFilters(this TemplateContext templateContext, FluidOptio
                 return filter.ProcessAsync(input, arguments, ctx);
             });
         }
+        if (options.AddFluidFilters)
+        {
+            templateContext.Options.Filters
+                .WithArrayFilters()
+                .WithStringFilters()
+                .WithNumberFilters()
+                .WithMiscFilters();
+        }
```

Comment:
Assuming there could be more filters, coming from Fluid and potentially other packages, a more extensible way would be to replace the options with a delegate that defaults to installing these filters, while allowing the application to override the delegate to e.g. reduce or increase the set of filters.

I can push that change if you want after merging the PR, just let me know.